src/localize.ts

```diff
@@ -49,21 +45,17 @@ export class Localize {
 
     const languageFilePath = resolve(rootPath, resolvedLanguage);
 
-    try {
-      const defaultLanguageBundle = JSON.parse(
-        resolvedLanguage !== defaultLanguage
-          ? readFileSync(resolve(rootPath, defaultLanguage), "utf-8")
-          : "{}"
-      );
+    const defaultLanguageBundle: Record<string, string> = JSON.parse(
+      resolvedLanguage !== defaultLanguage
+        ? readFileSync(resolve(rootPath, defaultLanguage), "utf-8")
+        : "{}"
+    );
 
-      const resolvedLanguageBundle = JSON.parse(
-        readFileSync(languageFilePath, "utf-8")
-      );
+    const resolvedLanguageBundle: Record<string, string> = JSON.parse(
+      readFileSync(languageFilePath, "utf-8")
+    );
 
-      return { ...defaultLanguageBundle, ...resolvedLanguageBundle };
-    } catch (err) {
-      throw err;
-    }
+    return { ...defaultLanguageBundle, ...resolvedLanguageBundle };
```

Comment:
why exception handling is removed?

Reply:
Same as https://github.com/shanalikhan/code-settings-sync/pull/1257#discussion_r495886441